syntax = "proto3";
package protos.playbooks;

import "protos/base.proto";

message ZendutyNotesWorkflowAction {
  int32 incident_number = 1;
}
